### ISSUE TITLE
Ccomments for the tutorial

### ISSUE THREAD

**Issue by Zireael** [diogosouza/logrocket-oauth2-example#1]
Hi,
I went through your oauth2 tutorial on [Logrocket](https://blog.logrocket.com/implementing-oauth-2-0-in-node-js/) and have 3 comments:
1) crypto package was deprecated and is now part of node. No need to put it as node_module dependency (you will get a warning if you do yarn install)
2) I had to add client_secret to get the user registration POST to work 
![image](https://user-images.githubusercontent.com/3856578/82064421-a2f5de00-96c4-11ea-8c08-5b546c4572aa.png)

2) Regarding the final part of the tutorial (By selecting the Authorization Tab, you get access to some interesting test features.), can you add a screenshot of what to put in the Postman fields?
I'm trying like so but no luck:

![image](https://user-images.githubusercontent.com/3856578/82064242-69bd6e00-96c4-11ea-8563-7d0e65ee807f.png)

Your tutorial might need some updates due to package upgrades, I guess?

Thanks for giving me a head start!

**Comment by Zireael** [comment]
Looks like this tutorial was based on an old tutorial here:
https://blog.cloudboost.io/how-to-make-an-oauth-2-server-with-node-js-a6db02dc2ce7
`bluebird` dependency is not required either

**Comment by diogosouza** [comment]
> Looks like this tutorial was based on an old tutorial here:
> https://blog.cloudboost.io/how-to-make-an-oauth-2-server-with-node-js-a6db02dc2ce7
> `bluebird` dependency is not required either

**Comment by diogosouza** [comment]
Hey @Zireael, how are you doing?

Yes, the second OAuth2 example, I took it as a base and adapted to the newest Ruby version, as well as some personal changes. 

Do you think it is a problem? I can change the example from the password grant to the client credentials, for example.

Let me know what you think.

**Comment by Zireael** [comment]
Hi, your tutorial is good, I'm happy you used PostgreSQL as I like the database too. But timestams said it was 3 weeks old and I was confused because it was using Bluebird (old promise polyfill) and crypto packages that are not needed anymore. I would keep it as is but maybe do yarn upgrade --latest and make it use latest package versions :) And if you could add example of how to use Postman to get new access token.

Do you know how to generate code and refresh_token? I saw a promising example [here](https://github.com/14gasher/oauth-example), but it doesn't work... :/

Was great learning opportunity, thanks!

**Comment by diogosouza** [comment]
Hey, I'm really sorry for the delay.. it's been really busy.

You're right, no need for them, I actually forgot to remove these dependencies from the final version. The other packages are actually already up to date.

I've submitted to LogRocket the changes.

Regarding the Postman testing features, it's simple. These are the settings to request a new token:

<img width="686" alt="Screen Shot 2020-05-27 at 15 51 53" src="https://user-images.githubusercontent.com/764655/83060817-6fa33f80-a032-11ea-871e-9a39b56e65ad.png">

Then, you click "Use token" and Postman will replace the "Access token" field with the generated one:

<img width="838" alt="Screen Shot 2020-05-27 at 15 52 05" src="https://user-images.githubusercontent.com/764655/83060910-8d70a480-a032-11ea-8ae6-821e7b731dfe.png">

Let me know if it works for you! :)

**Comment by JoshLDowns** [comment]
First off, thank you for the tutorial.  I'm new to Postgres and oAuth, and this was great. I just have one question. How would I set this up to fetch the current user info with just the bearer token in the header? The tutorial never went on to show grabbing the user from the data base and I was able to use the existing getUser function in the userDB to make it happen with the username and password, but wondering if you might know how I could make it work with just the access token in the header?
